ci/default-admin-admin.yaml

```diff
@@ -1,12 +0,0 @@
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: default-admin-admin
-subjects:
- - kind: ServiceAccount
```

Comment:
Do we not need an admin service account anymore? If so, then we can change the batch rule.

Reply:
I think this is dead.  I can't find where the account is used or created.  I'll update the batch rule.